rllib/core/learner/learner.py

```diff
@@ -1143,19 +1143,20 @@ def update(
             batch_iter = MiniBatchDummyIterator
 
         results = []
-        for minibatch in batch_iter(batch, minibatch_size, num_iters):
-            # Convert minibatch into a tensor batch (NestedDict).
-            tensor_minibatch = self._convert_batch_type(minibatch)
+        # Convert minibatch into a tensor batch (NestedDict).
```

Comment:
Can we add: "... on the correct device (e.g. GPU)"? This would clarify further.
Also note then that "we only perform copying to the correct device once so we do not have to move data in each minibatch iteration below".  something like this.

Reply:
done